libs/api-interfaces/src/lib/data-types/coscrad-data-type.enum.ts

```diff
@@ -14,6 +14,7 @@ export enum CoscradDataType {
      */
     CompositeIdentifier = 'COMPOSITE_IDENTIFIER',
     Year = 'YEAR',
+    Month = 'MONTH',
```

Comment:
No. Remove this. Month should be an API defined in the `api-interfaces`. See `MIMEType` defined in `api-interfaces`, for example.

Reply:
done